data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,270 @@ mission "A Sunracer Hovercraft Racing"
+		"You descend from your ship to enter the spaceport, "
+	word
+		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
+		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
+		"when a sharp laugh draws your attention to a small card game on a crate. "
+		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
+	word
+		"The dealer looks up and grins. "
+		"The dealer glances up, eyebrows arched. "
+		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "
+		"The dealer finally notices you, and his grin widens. "
+
+phrase "three card monte opening 2"
+	word
+		"He offers an easy challenge: "
+		"He spreads his arms over the cards in invitation: "
+		""
+		"He grandly gestures at the three waiting cards: "
+
+phrase "card anything but an ace"
+	word
+		"king"
+		"queen"
+		"jack"
+		"ten"
+		"two"
+	word
+		" of"
+
+phrase "card suits"
+	word
+		"hearts"
+		"spades"
+		"clubs"
+		"diamonds"
+
+mission "Three Card Monte"
+	minor
+	repeat
+	"offer precedence" 10
+	to offer
+		random < 20
+		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
+		credits > 1000
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
+		not attributes "deep" "station"
+	on offer
+		conversation
+			`${three card monte opening}`
+			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`
+				to display
+					"reputation: Pirate" > 100
+			`	${three card monte opening 2}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
+			action
+				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
+				"monte: not scam chance" >?= 60 # There's always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game
+			label "new game"
+			`	The dealer waits expectantly.`
+			choice
+				`	(Put down 1,000 credits to play the game.)`
+					to display
+						credits > 1000
+				`	(Pat my empty pockets. Can't play today.)`
+					decline
+					to display
+						credits < 1000
+				`	(Skip the game.)`
+					decline
+			label "new game start"
+			action
+				"monte: multiplication" = 1 + "roll: 2"
+				"monte: addition" = "roll: 3"
+				"monte: card spot" = ( "monte: multiplication" + "monte: addition" ) % 3
+				"monte: step" = 1
+				clear "monte: scammed"
+				payment -1000
+			`	You drop a 1,000 credit chip to play. "All you need to do is find the ace," the dealer says, flashing the ace of ${card suits} faceup towards you, then setting it down between two other facedown cards in the center. He starts shuffling with a practiced twist.`
```

Comment:
Again, in my mind card names are proper nouns (and StackExchange seems to agree with me). I'd probably capitalize all of these?

Reply:
Ditto to above, happy either way if anyone feels strongly.